recipes/nest/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+export MPI_FLAGS=--allow-run-as-root
+
+if [ $(uname) == Linux ]; then
+    export MPI_FLAGS="$MPI_FLAGS;-mca;plm;isolated"
+fi
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX \
+      -Dwith-mpi=OFF\
+      -Dwith-openmp=OFF \
+      -Dwith-python=3 \
+      -Dwith-gsl=$PREFIX \
+      -DREADLINE_ROOT_DIR=$PREFIX \
+      -DLTDL_ROOT_DIR=$PREFIX \
+      ..
+make -j2
+make install
+
+cp $PREFIX/lib64/* $PREFIX/lib -r
```

Comment:
Should you move these instead?
```suggestion
mv $PREFIX/lib64/* $PREFIX/lib/
```
Or better yet, pass `-DCMAKE_INSTALL_LIBDIR="${PREFIX}/lib"` to `cmake` if it's needed.

Reply:
Good idea. I will try

Reply:
I tried a lot and only 'cp' worked fine.